libvips/foreign/jxlsave.c

```diff
@@ -302,6 +302,24 @@ vips_foreign_save_jxl_process_output(VipsForeignSaveJxl *jxl)
 	return 0;
 }
 
+static int
+vips_foreign_save_jxl_get_delay(VipsForeignSaveJxl *jxl, int page_number)
+{
+	int delay;
+
+	if (jxl->delay &&
+		page_number < jxl->delay_length)
+		delay = jxl->delay[page_number];
+	else
+		// the old gif delay field was in centiseconds, so convert to ms
+		delay = jxl->gif_delay * 10;
+
+	/* Force frames with a small or no duration to 100ms for consistency
+	 * with web browsers and other transcoding tools.
+	 */
+	return VIPS_MIN(100, delay);
```

Comment:
To align with the other suggestion:
```suggestion
	return delay <= 10 ? 100 : delay;
```